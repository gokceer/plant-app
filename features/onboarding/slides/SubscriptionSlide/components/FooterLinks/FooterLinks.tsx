import React from "react";
import { StyleSheet, Text, View } from "react-native";

const FooterLinks = () => {
  return (
    <View style={styles.footer}>
      <Text style={styles.footerText}>Terms</Text>
      <Text style={styles.footerDot}>•</Text>
      <Text style={styles.footerText}>Privacy</Text>
      <Text style={styles.footerDot}>•</Text>
      <Text style={styles.footerText}>Restore</Text>
    </View>
  );
};

export default FooterLinks;

const styles = StyleSheet.create({
  footer: {
    flexDirection: "row",
    marginTop: 10,
    justifyContent: "center",
  },
  footerText: {
    color: "#888",
    marginHorizontal: 6,
    fontSize: 14,
  },
  footerDot: {
    color: "#888",
    fontSize: 14,
  },
});
