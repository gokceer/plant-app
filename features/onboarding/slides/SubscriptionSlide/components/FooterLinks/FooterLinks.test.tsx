import { render } from "@testing-library/react-native";
import React from "react";
import FooterLinks from "./FooterLinks";

describe("FooterLinks", () => {
  it("renders correctly and matches snapshot", () => {
    const { toJSON } = render(<FooterLinks />);
    expect(toJSON()).toMatchSnapshot();
  });
});
