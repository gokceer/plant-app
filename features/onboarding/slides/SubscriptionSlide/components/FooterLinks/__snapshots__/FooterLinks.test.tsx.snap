// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterLinks renders correctly and matches snapshot 1`] = `
<View
  style={
    {
      "flexDirection": "row",
      "justifyContent": "center",
      "marginTop": 10,
    }
  }
>
  <Text
    style={
      {
        "color": "#888",
        "fontSize": 14,
        "marginHorizontal": 6,
      }
    }
  >
    Terms
  </Text>
  <Text
    style={
      {
        "color": "#888",
        "fontSize": 14,
      }
    }
  >
    •
  </Text>
  <Text
    style={
      {
        "color": "#888",
        "fontSize": 14,
        "marginHorizontal": 6,
      }
    }
  >
    Privacy
  </Text>
  <Text
    style={
      {
        "color": "#888",
        "fontSize": 14,
      }
    }
  >
    •
  </Text>
  <Text
    style={
      {
        "color": "#888",
        "fontSize": 14,
        "marginHorizontal": 6,
      }
    }
  >
    Restore
  </Text>
</View>
`;
